dockerfiles/QAT/run.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+# NOTE: Some env variables are set in the Dockerfile - those that are
+# intended to be over-rideable.
```

Comment:
Might be worth adding `set -o nounset` to avoid surprises if the `Dockerfile` changes?

Also, how about adding the standard idiom to make debugging easier:

```sh
[ -n "${DEBUG:-}" ] && set -o xtrace
```

Reply:
`set -u` added.
There is already `-d` debug option on the command line - will that suffice?

Reply:
and might be worth commenting that this file is intended to run in a container, not in the host

Reply:
See other comment - it is in a subdir called `dockerfiles`, next to a file called `Dockerfile`..... seems fairly self explanatory...no?